packages/eslint-config-sentry-app/index.js

```diff
@@ -120,35 +120,45 @@ module.exports = {
       },
     ],
 
-    // Enforce a convention in module import order
-    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md
-    'import/order': [
+    /**
+     * Better import sorting
+     */
+    'sort-imports': 'off',
+    'import/order': 'off',
+    'simple-import-sort/imports': [
       'error',
       {
-        groups: ['builtin', 'external', 'internal', ['parent', 'sibling', 'index']],
-        'newlines-between': 'always',
-        pathGroups: [
-          {
-            pattern: '@emotion/styled',
-            group: 'external',
-          },
-          {
-            pattern: 'sentry*/**',
-            group: 'internal',
-            position: 'before',
-          },
-          {
-            pattern: 'getsentry*/**',
-            group: 'internal',
-            position: 'after',
-          },
-          {
-            pattern: 'admin*/**',
-            group: 'internal',
-            position: 'after',
-          },
+        groups: [
+          // Side effect imports.
+          ['^\\u0000'],
+
+          // Node.js builtins. You could also generate this regex if you use a `.js` config.
+          // For example: `^(${require("module").builtinModules.join("|")})(/|$)`
+          [
+            '^(assert|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|https|module|net|os|path|punycode|querystring|readline|repl|stream|string_decoder|sys|timers|tls|tty|url|util|vm|zlib|freelist|v8|process|async_hooks|http2|perf_hooks)(/.*|$)',
```

Comment:
Any reason we're not using the thing in the comment?

Reply:
Ah, was just copy pasting, we could use it since we assume everyone's on the same version of node